packages/librewolf-deb/librewolf-deb.pacscript

```diff
@@ -1,7 +1,7 @@
 pkgname="librewolf-deb"
 gives="librewolf"
 replaces=("${gives}")
-pkgdesc="A fork of Firefox, focused on privacy, security and freedom"
+pkgdesc="A fork of Firefox, focused on privacy, security and freedom."
```

Comment:
```suggestion
pkgdesc="A fork of Firefox, focused on privacy, security and freedom"
```
The lack of a dot is intended

Reply:
Is that mean it doesn't need dots in end word for description?

Reply:
Yeah we avoid dots at the end of `pkgdesc`